### PR TITLE
Add ability to send application properties

### PR DESCRIPTION
**Pull request by gpad**

### CODE REVIEW

File: src/amqp10/encoder.ts [hunk 1]
```diff
@@ -41,6 +50,41 @@ function lengthOfProperties(properties?: MessageProperties) {
...
+    return 0
+  }
+
+  // var size = DescribedFormatCode.Size
+  // size += sizeof(byte) //FormatCode.List32
+  // size += sizeof(uint) // field numbers
+  // size += sizeof(uint) // PropertySize
+  // size += MapSize()
+  // return size
+  return 3 + 1 + 4 + 4 + getApplicationPropertySize(applicationProperties)
```
Comment (Yattol) [inline src/amqp10/encoder.ts:64]: this `3 + 1 + 4 + 4 + something` is starting to appear in multiple places, should we consider using a const?

**Comment by Yattol** [comment]
LGTM, just a small comment/suggestion, feel free to ignore it

**Review by Yattol — APPROVED** [review]

**Comment by albertobarrila** [comment]
LGTM

**Review by albertobarrila — APPROVED** [review]
LGTM

**Pull request merged**